### ISSUE TITLE
Wrong gradient accumulate!

### ISSUE THREAD

**Issue by crj1998** [huaishuiweizhu/stylegan2-tiny#1]
https://github.com/huaishuiweizhu/stylegan2-tiny/blob/e4537803b4ffb891c7ce0fb5a20a83829e3278b3/train.py#L141-L154

This implentation repeat train the same data with accumulation steps. I think you misunderstood the gradient accumulation.

**Comment by GuichenSociety** [comment]
> https://github.com/huaishuiweizhu/stylegan2-tiny/blob/e4537803b4ffb891c7ce0fb5a20a83829e3278b3/train.py#L141-L154
> 
> This implentation repeat train the same data with accumulation steps. I think you misunderstood the gradient accumulation.

Yes, I am a self-taught beginner. I have some understanding of convolutions, activation functions, and optimizers in terms of code, but I am not very familiar with backpropagation gradients. In this regard, I mainly rely on the experts to learn. Thank you very much for pointing out the errors, but how can I correct them? If it is convenient, can you add a WeChat account (this is a Chinese application) or send it to my email 2304617303@qq.com. I really look forward to your reply
